src/main/resources/application.properties

```diff
@@ -149,6 +149,19 @@ spring.elasticsearch.jest.read-timeout=10000ms
 management.health.elasticsearch.enabled=false
 management.health.status.order=DOWN, OUT_OF_SERVICE, UNKNOWN, UP, OUT_OF_SYNC, PROBLEM, COULD_NOT_CONNECT
 
+# Elasticsearch mappings
+elasticsearch.type-mappings.defaultText=Text, TextArea, FixedList, FixedListEdit, MultiSelectList, FixedRadioList, DynamicList
+elasticsearch.type-mappings.defaultDouble=Number, MoneyGBP
+elasticsearch.type-mappings.defaultDate=Date, Time, DateTime
+elasticsearch.type-mappings.ccdPhoneUK=PhoneUK
+elasticsearch.type-mappings.defaultKeyword=YesOrNo, Email, Postcode
+elasticsearch.type-mappings.ccdDocument=Document
```

Comment:
why are these needed? I thought we only needed to know whether a CCD type is a defaultText